src/libs/actions/Policy/Policy.ts

```diff
@@ -2600,6 +2601,78 @@ function togglePolicyUberAutoRemove(policyID: string | undefined, enabled: boole
     API.write(WRITE_COMMANDS.POLICY_UBER_AUTO_REMOVE, params, {optimisticData, successData, failureData});
 }
 
+/**
+ * Invites workspace employees to Uber for Business
+ */
+function inviteWorkspaceEmployeesToUber(policyID: string, emails: string[]) {
+    if (!policyID || !emails || emails.length === 0) {
```

Comment:
`!emails` does not make sense, since it will never be true:
```suggestion
    if (!policyID || emails.length === 0) {
```

Reply:
I will push in a moment, resolved not to lose what left to fix